src/elevator/fsm.rs

```diff
@@ -74,14 +76,21 @@ pub struct ElevatorFSM {
     fsm_order_complete_tx: cbc::Sender<(u8, u8)>,
     fsm_state_tx: cbc::Sender<ElevatorState>,
 
+    //Network channels
+    _net_peer_tx_enable_tx: cbc::Sender<bool>,
+
     // Private fields
     fsm_terminate_rx: cbc::Receiver<()>,
     hall_requests: Vec<Vec<bool>>,
     state: ElevatorState,
     n_floors: u8,
     obstruction: bool,
+    started_moving: bool,
```

Comment:
Do we need all the extra variables? Or is it possible to do the same as in the door timer to keep it uniform?